package.json

```diff
@@ -16,7 +16,9 @@
     "onCommand:mdb.connectWithURI",
     "onCommand:mdb.addConnection",
```

Comment:
Can we have several active connections at the same time? For me `addConnection`sounds like we are already connected to one server and want to add one more connection to the existing one.

Reply:
Only 1 active connection at a time, but we can have a few saved connection configurations. I think `connect` and `connectWithURI` is probably best - good point. I'll update them to that.